### DISCUSSION TITLE
Help with old deleted large objects

### DISCUSSION THREAD

**Discussion by lacostenycoder** [git-lfs/git-lfs#5221] [Q&A]
Should I still migrate to LFS if all the large objects in the original repo have since be deleted from all current branches?  I guess what I'm wondering is what is the point of maintaining those deleted large files in the new LFS version of the repository?  Furthermore, if LTS is ultimately creating a new mirror, is there point to using it now that the large files are being "purged"?

**Comment by bk2204** [comment]
Hey,

It really depends.  If you have a large amount of old large files and they don't delta well, you could end up saving several gigabytes or more of repository space, which might be very valuable to you and other contributors to the repository.  However, if there's just a handful of large files, they're all deleted, and you aren't adding new ones, then it may not be worth it.

If you're in the latter case and you're trying to migrate to GitHub, you may ask Support whether that's okay, and sometimes they'll grant an exception to the 100 MB file limit for the initial push.

**Reply by lacostenycoder** [reply]
Oh, that last bit of advice sounds good thanks! I'll try it.
